devbox.json

```diff
@@ -4,7 +4,7 @@
     "clusterctl@latest",
     "docker@latest",
     "envsubst@latest",
-    "go@1.23.3",
+    "go@1.24.0",
```

Comment:
nix should have 1.24.1 for the latest version

Reply:
thanks, updated.